src/groups/mqb/mqbblp/mqbblp_queueenginetester.h

```diff
@@ -642,37 +641,21 @@ QueueEngineTester::createQueueEngineHelper(mqbi::QueueEngine* engine)
 
 // MANIPULATORS
 template <typename T>
-inline mqbi::QueueEngine* QueueEngineTester::createQueueEngine()
+inline T* QueueEngineTester::createQueueEngine()
 {
     // PRECONDITIONS
     BSLS_ASSERT_OPT(!d_queueEngine_mp && "'createQueueEngine()' was called");
-
+    T* result;
     // Create and configure Queue Engine
-    d_queueEngine_mp.load(new (*d_allocator_p)
+    d_queueEngine_mp.load(result = new (*d_allocator_p)
```

Comment:
Could we put this `result =` up into the initializer above?  Just to make it 100% clear later on that the assignment here is intentional?

Reply:
It should be refactored, of course